.moban.dt/coala-multi-setup.py.jj2

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
```

Comment:
can you submit this as a change to https://gitlab.com/coala/mobans/ , using yaml variables or  `package_module` to control the new functionality.

Reply:
MR ready https://gitlab.com/coala/mobans/merge_requests/29

Reply:
merged; this isnt needed any more?